script.js

```diff
@@ -263,3 +269,45 @@ function closeModal() {
   pageBody.style.height = '';
   pageBody.style.overflowY = '';
 }
+
+// Implement local storage
+
+const userName = document.getElementById('user-name');
+const message = document.getElementById('message');
+const formInputs = {
+  userName: '',
+  emailInput: '',
+  message: '',
+};
+
+function populateStorage() {
+  formInputs.userName = userName.value;
+  formInputs.emailInput = emailInput.value;
+  formInputs.message = message.value;
+  localStorage.setItem('form', JSON.stringify(formInputs));
+}
+
+function populateForm() {
+  const storedForm = JSON.parse(localStorage.getItem('form'));
+  userName.value = storedForm.userName;
+  emailInput.value = storedForm.emailInput;
+  message.value = storedForm.message;
+}
+
+if (!localStorage.getItem('form')) {
+  populateStorage();
+} else {
+  populateForm();
+}
```

Comment:
Why use a negative here? you could say 
```suggestion
if (localStorage.getItem('form')) {
  populateForm();
} else {
  populateStorage();
}
```